src/models/settlement/facade.js

```diff
@@ -39,7 +39,7 @@ const Utility = require('../../handlers/lib/utility')
 const getNotificationMessage = function (action, destination, payload) {
   return {
     id: Uuid(),
-    from: Enums.headers.FSPIOP.SWITCH,
+    from: Config.HUB_NAME,
```

Comment:
Please, also update line 49 below. Probably it's also good idea to remove the export from enums.js, so no one accidentally uses it again.

Reply:
I wasn't sure about line 49 is it the name or something else?

Reply:
It's basically the same property: Enums.headers.FSPIOP.SWITCH, which is Yes

Reply:
In case we need to preserve both, I'd suggest adding everything to Config.HUB and refer to those. E.g. **Config.HUB.NAME** and **Config.HUB.FSPIOP**. Otherwise I feel it would induce further misunderstanding and misuse.

Reply:
Its not the same property. One is Hub participant name, the other is something else. 
all FSPIOP related values are in the ENUMS. If we need to have consistency we should rework of some code, I guess.